lib/ansible/modules/cloud/digital_ocean/digital_ocean_droplet.py

```diff
@@ -0,0 +1,303 @@
+version_added: "2.5"
+author: "Gurchet Rai (@gurch101)"
+options:
+  state:
+    description:
+     - Indicate desired state of the target.
+    default: present
+    choices: ['present', 'absent']
+  id:
+    description:
+     - Numeric, the droplet id you want to operate on.
+  name:
+    description:
+     - String, this is the name of the droplet - must be formatted by hostname rules, or the name of a SSH key.
+  unique_name:
+    description:
+     - Bool, require unique hostnames.  By default, DigitalOcean allows multiple hosts with the same name.  Setting this to "yes" allows only one host
+       per name.  Useful for idempotence.
+    default: "no"
+    choices: [ "yes", "no" ]
+  size:
+    description:
+     - This is the slug of the size you would like the droplet created with.
+  image:
+    description:
+     - This is the slug of the image you would like the droplet created with.
+  region:
+    description:
+     - This is the slug of the region you would like your server to be created in.
+  ssh_key_ids:
+    description:
+     - Optional, array of SSH key (numeric) ID that you would like to be added to the server.
+  virtio:
+    description:
+     - Bool, turn on virtio driver in droplet for improved network and storage I/O.
+    default: "yes"
+    choices: [ "yes", "no" ]
+  private_networking:
+    description:
+     - Bool, add an additional, private network interface to droplet for inter-droplet communication.
+    default: "no"
+    choices: [ "yes", "no" ]
+  backups_enabled:
+    description:
+     - Optional, Boolean, enables backups for your droplet.
+    default: "no"
+    choices: [ "yes", "no" ]
+  user_data:
+    description:
+      - opaque blob of data which is made available to the droplet
+    required: false
+    default: None
+  ipv6:
+    description:
+      - Optional, Boolean, enable IPv6 for your droplet.
+    required: false
+    default: "no"
+    choices: [ "yes", "no" ]
+  wait:
+    description:
+     - Wait for the droplet to be active before returning.  If wait is "no" an ip_address may not be returned.
+    default: "yes"
+    choices: [ "yes", "no" ]
+  wait_timeout:
+    description:
+     - How long before wait gives up, in seconds, when creating a droplet.
+    default: 300
+  backups:
+    description:
+     - Optional, Boolean, indicates whether automated backups should be enabled.
+    default: "no"
+    choices: [ "yes", "no" ]
+  tags:
+    description:
+     - Optional, List, A list of tag names as strings to apply to the Droplet after it is created. Tag names can either be existing or new tags.
+  volumes:
+    description:
+     - Optional, List, A list including the unique string identifier for each Block Storage volume to be attached to the Droplet.
+  oauth_token:
+    description:
+     - DigitalOcean OAuth token. Can be specified in DO_API_KEY or DO_API_TOKEN environment variables
+    required: true
+requirements:
```

Comment:
Please include the `monitoring` option as well. This will allow users to create droplets with [enhanced monitoring](https://developers.digitalocean.com/documentation/v2/#create-a-new-droplet) from the beginning instead of forcing them to add the solution later on.

Reply:
Looks like `monitoring:` has been added.